drivers/boston_project_scorecard/app/models/boston_project_scorecard/report.rb

```diff
@@ -189,16 +189,15 @@ def run_and_save!
 
       if apr.present?
         # Project Performance
-        # 1a / 1b
-        assessment_answers.merge!(rrh_exits_to_ph: percentage(answer(apr, 'Q23c', 'B46'))) if rrh?
-        if psh?
-          value = percentage((answer(apr, 'Q5a', 'B1') - answer(apr, 'Q23c', 'B43') + answer(apr, 'Q23c', 'B44')) /
-            (answer(apr, 'Q5a', 'B1') - answer(apr, 'Q23c', 'B45')).to_f)
-          assessment_answers.merge!(psh_stayers_or_to_ph: value)
-        end
+        one_a_value = percentage(answer(apr, 'Q23c', 'B46'))
+        one_b_value = percentage((answer(apr, 'Q5a', 'B1') - answer(apr, 'Q23c', 'B43') + answer(apr, 'Q23c', 'B44')) /
+          (answer(apr, 'Q5a', 'B1') - answer(apr, 'Q23c', 'B45')).to_f)
```

Comment:
might `(answer(apr, 'Q5a', 'B1') - answer(apr, 'Q23c', 'B45')).to_f)` ever be zero?

Reply:
It might -- but `x / 0.0` returns `Infinity` which is dealt with by `percentage` -- seemed cleaner than throwing `rescue`s around